app/middleware/setLocals.js

```diff
@@ -9,11 +9,8 @@ function fromRequest(req, res, next) {
     latitude: req.query.latitude,
     longitude: req.query.longitude,
   };
-  const fullUrl = fullUrlUtils(req);
   const backLink = backLinkUtils(req);
-  res.locals.fullUrl = {
-    href: fullUrl.url,
-  };
+  res.locals.completeOriginalUrl = completeOriginalUrl(req);
```

Comment:
Can this line be moved above the creation of the const? It's a bit odd to have it in the middle of a variable declaration and the use of that variable.

Reply:
Done